# HG changeset patch
# User Rik <rik@octave.org>
# Date 1471993796 25200
#      Tue Aug 23 16:09:56 2016 -0700
# Node ID 3dc8764e3bd55f1d5935097bdd2e12d72230b3e5
# Parent  459545bc901930e068e3046715f5fe774f13061b
colorbar.m: Fix typo of double quote for transpose operator.

* colorbar.m: Fix typo of double quote for transpose operator.

diff --git a/scripts/plot/draw/colorbar.m b/scripts/plot/draw/colorbar.m
--- a/scripts/plot/draw/colorbar.m
+++ b/scripts/plot/draw/colorbar.m
@@ -486,17 +486,17 @@ endfunction
 %! n = 64; x = kron (1:n, ones (n,1)); x = abs (x - x.');
 %! imagesc (x);
 %! colorbar ();
 %! title ("colorbar() example");
 
 %!demo
 %! clf;
 %! colormap ("default");
-%! n = 64; x = kron (1:n, ones (n,1)); x = abs (x - x.");
+%! n = 64; x = kron (1:n, ones (n,1)); x = abs (x - x.');
 %! imagesc (x);
 %! colorbar ("westoutside");
 
 %!demo
 %! clf;
 %! colormap ("default");
 %! n = 64; x = kron (1:n, ones (n,1)); x = abs (x - x.');
 %! imagesc (x);
